### ISSUE TITLE
Inconsist indent rule behaviour

### ISSUE THREAD

**Issue by beenotung** [typescript-eslint/typescript-eslint#1535]
**What code were you trying to parse?**

```ts
export class Store<ALongGenericTypeName> {
  private [Symbol.aLongSymbolName] = new Map<
    string,
    ALongGenericTypeName
  >();
}
```

**What did you expect to happen?**
Above code is already indented correctly

**What actually happened?**
The eslint complains, and suggest to change to:
```ts
export class Store<ALongGenericTypeName> {
  private [Symbol.aLongSymbolName] = new Map<
  string,
  ALongGenericTypeName
  >();
}
```

**Versions**

| package                                   | version |
| ----------------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin-tslint` | `2.18.0` |
| `@typescript-eslint/parser`               | `2.18.0` |
| `TypeScript`                              | `3.7.5` |
| `ESLint`                                  | `6.8.0` |
| `node`                                    | `12.13.1` |
| `npm`                                     | `6.13.6` |
| `pnpm`                                     | `4.7.2` |

**Comment by bradzacher** [comment]
Duplicate of #455
